web/pgadmin/browser/server_groups/servers/databases/schemas/tables/columns/utils.py

```diff
@@ -474,3 +475,12 @@ def parse_column_variables(col_variables):
             k, v = spcoption.split('=')
             spcoptions.append({'name': k, 'value': v})
     return spcoptions
+
+
+def fetch_geometry_details(data):
+    if 'typname' in data and (data['typname'] in ('geometry', 'geography')):
+        match_obj = re.search(r'([a-zA-Z]+),(\d+)', data['cltype'])
```

Comment:
looks like this logic is repeating in columns as well. We can have something like parseArgs which will return an array of args.

Reply:
Regular expression in this case is different. However, I can check if we can have a common function.